pkgs/applications/networking/irc/tiny/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, rustPlatform, fetchpatch, fetchFromGitHub
+, pkg-config, dbus, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tiny";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "osa1";
+    repo = pname;
+    rev = version;
+    sha256 = "044l68pcm22192n57h0zrgjsfgm820ics9p6ff7axfq2vi0j98fr";
+  };
+
+  cargoSha256 = "044l68pcm22192n57h0zrgjsfgm820ics9p6ff7axfq2vi0j98fr";
+
+  RUSTC_BOOTSTRAP = 1;
+
+  nativeBuildInputs = stdenv.lib.optional stdenv.isLinux pkg-config;
+  buildInputs = stdenv.lib.optionals stdenv.isLinux [ dbus openssl ];
+
+  meta = with stdenv.lib; {
```

Comment:
there's a `lib` alias at the package scope, you could add `lib` to the parameters and replace `stdenv.lib` by just `lib`. it would make these lines slightly shorter